pkg/settings/registry.go

```diff
@@ -119,6 +119,7 @@ var retiredSettings = map[string]struct{}{
 	"trace.datadog.project":                                            {},
 	"sql.defaults.interleaved_tables.enabled":                          {},
 	"sql.defaults.copy_partitioning_when_deinterleaving_table.enabled": {},
+	"sql.ttl.range_batch_size":                                         {},
```

Comment:
nit: rebase and put it under the "removed as oof 22.2" comment

Reply:
Hmm that won't backport nicely, technically this is straight away in 22.1.ill put something in between